src/main/java/org/veupathdb/service/eda/ds/plugin/standalonemap/BubbleMapMarkersPlugin.java

```diff
@@ -152,10 +152,12 @@ protected void writeResults(OutputStream out, Map<String, InputStream> dataStrea
       MarkerAggregator<Double> aggregator = data.getMarkerAggregator();
       if (aggregator != null) {
         Double aggregation = aggregator.finish();
-        if (aggregation != null)
+        if (aggregation != null) {
           mapEle.setOverlayValue(overlayConfig.map(oc -> oc.serializeAverage(aggregation)).orElse(null));
+	  // only output marker data where there are overlay values (issue #334)
+	  output.add(mapEle);
```

Comment:
Oh, so if there's no value for this, we don't want to give any of the other info either?

Reply:
Aha, caught up on this in slack and see why we want it this way.  Approved and merged.

Reply:
I think so. The (existence of) numerator/denominator values form part of the "little filter" that restricts what markers are shown.

And the front end is expecting a value!